components/automate-ui/src/app/modules/infra-proxy/environments/environments.component.html

```diff
@@ -4,15 +4,25 @@
   <ng-container *ngIf="!environmentsListLoading && !authFailure">
     <chef-table>
       <chef-thead>
-        <chef-tr>
-          <chef-th>Environments Name</chef-th>
+        <chef-tr class="no_border_tr">
+          <chef-th class="no_border_th">Name</chef-th>
+          <chef-th class="no_border_th">Description</chef-th>
+          <chef-th class="no_border_th"></chef-th>
+          <chef-th class="no_border_th"></chef-th>
+          <chef-th class="no_border_th"></chef-th>
         </chef-tr>
       </chef-thead>
       <chef-tbody>
         <chef-tr *ngFor="let environment of environments">
           <chef-td>
             <a [routerLink]="['/infrastructure','chef-servers', serverId, 'organizations', orgId, 'environments', environment.name]">{{ environment.name }}</a>
           </chef-td>
+          <chef-td>
+            {{ environment.description }}
```

Comment:
With long description the text is getting wrapped although we have enough space to the right. 
![image](https://user-images.githubusercontent.com/7055118/107897091-429b1200-6f5e-11eb-8149-3a48a87af035.png)
Is this how the text should look like?